platforms/android/build_sdk.py

```diff
@@ -390,6 +390,10 @@ def get_ndk_dir():
         os.environ['ANDROID_NDK_HOME'] = os.environ["ANDROID_NDK"]
         show_samples_build_warning = True
 
+    android_sdkmanager = os.path.join(os.environ['ANDROID_SDK'], 'tools', 'bin', 'sdkmanager')
+    accept_licenses_proc = subprocess.Popen(['yes | ' + android_sdkmanager + ' --licenses'], shell=True)
+    log.debug('yes | ' + android_sdkmanager + ' --licenses')
```

Comment:
1. This must be done by User before running this script. At least not implicitly in automatic mode.
2. This would not work on Windows.

Reply:
@alalek I wasn't intending to leave this code in. It was just an attempt to get the failing build to pass. (Which anyway doesn't seem to have worked.) I will revert it.